addons/website/views/website_templates.xml

```diff
@@ -1060,7 +1060,7 @@ Sitemap: <t t-esc="url_root"/>sitemap.xml
 </template>
 
 <template id="website_search_box" name="Website Searchbox">
-    <div class="input-group">
+    <div t-attf-class="input-group #{classes if classes else ''}">
```

Comment:
Don't forget to review all of those before review ;) (`#{classes}` produces the same result and even a better one if there was not the `input-group` class ;))